.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: ruby
 
+cache: bundler
+
 bundler_args: --without documentation
 
 rvm:
-  - 2.6.2
-  - 2.5.5
-  - 2.4.5
-  - 2.3.8
+  - 2.7
+  - 2.6
+  - 2.5
+  - 2.4
+  - 2.3
   - jruby-9.2.6.0
-
-sudo: false
```

Comment:
Not needed anymore. https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration